lib/rpc.ts

```diff
@@ -160,6 +160,12 @@ export class Rpc extends EventEmitter {
       // TODO Test, then explain how this works.
       return new Proxy({}, {
         get: (target, property: string, receiver) => {
+          if (property === "then") {
+            // By default, take care not to look "thenable":
```

Comment:
Maybe add "so that the stub can be returned as a value of a Promise" (to make it clearer that this is about the stub rather than about the methods in the stub).